src/main/java/com/esri/samples/na/closest_facility/ClosestFacilitySample.java

```diff
@@ -158,19 +165,14 @@ private void createFacilitiesAndGraphics() {
 
     // for each facility, create a graphic and add to graphics overlay
     facilities.stream().map(f -> new Graphic(f.getGeometry(), facilitySymbol))
-        .collect(Collectors.toCollection(() -> graphicsOverlay.getGraphics()));
+        .collect(Collectors.toCollection(() -> facilityGraphicsOverlay.getGraphics()));
   }
 
   /**
    * Adds facilities(hospitals) and user's incident(black cross) to closest facility parameters which will be used to 
    * display the closest route from the user's incident to its' nearest facility.
    */
   private void populateParametersAndSolveRoute() {
-    // clear any parameters that were set
-    facilityParameters.clearFacilities();

```

Comment:
Not for sure why these clear methods even exist since anytime you set facilities or incidents is clears all previous ones anyways.